src/cr/cube/slices.py

```diff
@@ -1090,6 +1094,11 @@ def __init__(self, slice_, subtotal):
         self._slice = slice_
         self._subtotal = subtotal
 
+    @lazyproperty
+    def hidden(self):
```

Comment:
Should be `.is_hidden` I think, to be consistent with existing properties using that name, like in `_NewElement` for example.

Reply:
I feel you... But I like the sound of it in a list comprehension, or a generator expression:
`[row for row in self._slice.rows if not row.hidden]`
in my head - it sounds more like English - "do this, if not hidden..." then "do this if not is hidden" (which sounds more like a Spanish guy trying to transliterate Spanish into English).

We can decide on what we want to use and stick with it through the `cr.cube`. This is just a _slight_ preference, not a strong one.